File: client-domain/src/main/java/com/jraska/github/client/common/Maps.java
package com.jraska.github.client.common;

import java.util.HashMap;
import java.util.Map;

public final class Maps {
  private Maps() {
    Preconditions.throwNoInstances();
  }

  public static <K, V> Map<K, V> newHashMap(K key, V value) {
    HashMap<K, V> map = new HashMap<>();
    map.put(key, value);
    return map;
  }
}

File: client-domain/src/main/java/com/jraska/github/client/common/Maps.kt
package com.jraska.github.client.common

import java.util.*

class Maps private constructor() {
  companion object {

    fun <K, V> newHashMap(key: K, value: V): Map<K, V> {
      val map = HashMap<K, V>()
      map.put(key, value)
      return map
    }
  }
}
